server/kv.go

```diff
@@ -536,6 +537,9 @@ func (store *store) DeleteInstance(id types.ID) error {
 func (store *store) LoadInstances() (*Instances, error) {
 	kv := kvstore.NewStore(kvstore.NewPluginStore(store.plugin.API))
 	vs, err := kv.ValueIndex(keyInstances, &instancesArray{}).Load()
+	if errors.Cause(err) == kvstore.ErrNotFound {
+		return &Instances{}, nil
```

Comment:
There is a panic happening due to this `Instances` not being properly initialized. Can we change this line to the following?

```suggestion
		return NewInstances(), nil
```